### PR TITLE
add(epp): hms ewon snmp monitoring proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description

-

## Target serie

- [X] 20.04.x
- [X] 20.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+id: hardware-devices-hms-ewon-snmp
+title: HMS Ewon SNMP
+---
+
+## Plugin-Pack Assets
+
+### Monitored Objects
+
+The plugin-pack HMS Ewon including monitoring of Tags.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:10]: Uppercase Plugin-Pack

Comment (Sims24) [reply en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:10]: ing form misused

File: en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+| interface status           | Status of the tag  |      |
+| *tagname*#tag.value.count  | Tag value          |      |
+
+It is possible to filter on the name of a tag using a REGEXP of the form [```--filter-tag-name='Fuel'```].
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To control your HMS Ewon, the SNMP must be configured.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:29]: Add some public manufacturer documentation if any

File: en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+Expected command output is shown below: 
+
+```bash
+CRITICAL: Tag 'Compt_H_Partiel' value: 441985 | 'Compt_H_Partiel#tag.value.count'=441985;0:40000;0:50000;; 'Compt_Ea_Partiel#tag.value.count'=20507;0:40000;0:50000;; 'Compt_Er_Partiel#tag.value.count'=1040;0:40000;0:50000;;
+Tag 'Compt_H_Partiel' status: none, value: 441985
+Tag 'Compt_Ea_Partiel' status: none, value: 20507
+Tag 'Compt_Er_Partiel' status: none, value: 1040
+```
+
+The command above monitors a HMS Ewon tags (```--plugin=hardware::devices::hms::ewon::snmp::plugin --mode=tags```) identified
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:103]: remove 'a'

File: fr/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+---
+id: hardware-devices-hms-ewon-snmp
+title: HMS Ewon SNMP
+---
+
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:5]: Uppercase Plugin-Pack

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md (hunk 2 above)
Comment (garnier-quentin) [reply en/integrations/plugin-packs/procedures/hardware-devices-hms-ewon-snmp.md:29]: i don't have documentation link

**Review by Sims24 — APPROVED** [review]

**Pull request merged**